packages/f/foomatic-db-engine/package.yml

```diff
@@ -13,8 +14,9 @@ builddeps  :
 rundeps    :
     - foomatic-db
 setup      : |
-    %configure
+    %configure_no_runstatedir
 build      : |
     %make
 install    : |
     %make_install
+    rm -rf $installdir/etc
```

Comment:
What is this for?

Reply:
Hmm, I added it to remove this path `<Path fileType="config">/etc/foomatic</Path>` as we don't install to `/etc` right ? Now, I looked at the git history, this package has been installing something to `/etc` since [included](https://github.com/getsolus/packages/commit/d19ae8a6921da61e9ce92a572a77edfa9368b357#diff-84e6df7959551e8fa0cb6badf3642594a48848b1709f91d43ba7a92e1afa1811R29). Will look into this further.

Reply:
It is an empty folder directory. I think it was safe to remove. Marked this as resolved